packages/dialtone-vue2/package.json

```diff
@@ -111,7 +111,7 @@
   },
   "peerDependencies": {
     "@dialpad/dialtone-css": "^7.30.0 || ^8.21.2",
-    "@dialpad/dialtone-icons": "^4.3.0",
+    "@dialpad/dialtone-icons": "latest",
```

Comment:
We had a lot of problems with latest before is it safe to use here?

Reply:
I think it's safe as we're no longer using `@dialpad/dialtone-vue` package on product and this will only affect that specific package.

I think it might benefit anyone having the dialtone-vue package as it'd always have the latest icons, right?

Reply:
I'm thinking about the implications of setting this here.. I think it might cause some issues with deployment stability

If we set the peer dependency to latest, that means it will automatically update to latest upon npm install. This means if they do a beta cut for the week and then we do a new release of dialtone-icons, it will either start erroring out here saying the peer dependency is incorrect, or it will automatically update dialtone-icons to the latest next time they do a build on beta. This would be bad because we could potentially introduce a bug in beta without actually updating dependencies

Reply:
> I think it's safe as we're no longer using `@dialpad/dialtone-vue` package on product and this will only affect that specific package.

Doesn't our monopackage also reference package jsons from other packages? It would have to right?

Reply:
Not really, it if was that way we wouldn't need to manually define exports.

The dependencies, devDependencies and peerDependencies are independent and isolated on each package. I'm pretty sure that peer dependency it's not going to affect the mono package.

But it makes sense to change that to `workspace:*` instead, that way it'd be the latest version number we released on purpose, are you ok with that approach?